include/wx/event.h

```diff
@@ -48,6 +48,7 @@ class WXDLLIMPEXP_FWD_BASE wxEventFilter;
     class WXDLLIMPEXP_FWD_CORE wxReadOnlyDC;
     class WXDLLIMPEXP_FWD_CORE wxWindow;
     class WXDLLIMPEXP_FWD_CORE wxWindowBase;
+    class WXDLLIMPEXP_FWD_CORE wxMenuItem;
```

Comment:
nitpick: The classes are declared in the alphabetical order, so `wxMenuItem` should be moved after `wxMenu`.

Reply:
Thank you !